packages/apps-config/src/ui/general/index.ts

```diff
@@ -43,7 +44,8 @@ const chainColors: Record<string, any> = [
   ['darwinia parachain', chainRoccoDarwinia],
   ['laminar turbulence pc1', laminarRoccoAcala],
   ['Westend', chainWestend],
-  ['phala poc 2', chainPhala]
+  ['phala poc 2', chainPhala],
+  ['crust maxwell cc2', chainCrust]
```

Comment:
Just move it up to before Darwininia

Reply:
Yes, I missed the one just before this on review, so would need to do that in a different PR :(